CMakeLists.txt

```diff
@@ -107,6 +107,21 @@ endif()
 # produce compile_commands.json
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+if (FLEX_VERSION VERSION_LESS 2.6.0)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Dregister=")
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Dregister=")
```

Comment:
@albert-github Isn't it enough to set `CMAKE_CXX_FLAGS` and not all the build type specific settings as well? This [post](https://stackoverflow.com/a/60622717/784672) seems to suggest so.

Reply:
Indeed looks like not to be necessary. `CMAKE_CXX_FLAGS_...` are supplementary to `CMAK_CXX_FLAGS`.